### PR TITLE
Plot history of results in UI 

### PR DESCRIPTION
**Pull request by leplatrem**
* [x] Play with plotly
* [ ] Add permissions for Redash to be able to read BigQuery entries
* [x] Fetch history of values from Redash and serve in result endpoint
* [x] Adjust apparence (dark/light mode)
* [x] Add tests about history fetching
* [x] Refresh behaviour
* [x] Responsive behaviour

Might require to do #362 first...

### CODE REVIEW

**Comment by leplatrem** [comment]
Screenshot from 2020-11-12 10-34-50

**Comment by leplatrem** [comment]
Screenshot from 2020-11-12 11-14-17
Screenshot from 2020-11-12 11-14-03

File: poucave/html/js/app.js [hunk 1]
```diff
@@ -698,6 +712,128 @@ class Check extends Component {
...
+    // Yellow lines with history of values.
+    const scalarPlot = {
+      x: [],
+      y: [],
+      mode: 'lines',
+      fill: 'tonexty',
+      fillcolor: '#ffb70030',
+      line: {
+        shape: 'hvh',
+        color: "#ffb700",
```
Comment (rehandalal) [inline poucave/html/js/app.js:759]: I wonder if we should move all the colors here and below into constants that we can edit easily in one place?

**Review by rehandalal — APPROVED** [review]
looks good to me! nice work!! just one suggestion for colors. 

r+wc

File: poucave/html/js/app.js (hunk 1 above)
Comment (leplatrem) [reply poucave/html/js/app.js:759]: yeah will do, thanks!

**Pull request merged**